### DISCUSSION TITLE
Failed dkim signature

### DISCUSSION THREAD

**Discussion by Emlock93** [modoboa/modoboa#2584] [Q&A]
Hello,

I managed to install modoboa server on my oracle vps arm instance,  Everything is fine except that i cannot make the dkim signature work.
if i go to the admin interface and select my domain i see that the dkim icon is Yellow saying that the record is present on my dns registrar but The signature is not valid.
In fact if i send an email to my Gmail address i see that the dkim fails, also if go to
Mailtester.com i got a miserabile 3.5/10 score saying that my dkim sign is not valid.
Strange fact is that if i verify my dkim record on mxtoolbox it says that Everything is good.
I've also tried to recreate my domain on modoboa using a 4096bit key but The situation didnt change.
I dont know what to do to make this work.

Thanks

**Comment by tonioo** [comment]
Have you checked if OpenDKIM is running?
